### ISSUE TITLE
failed to precompile dart_style:format:

### ISSUE THREAD

**Issue by rayk** [dart-lang/dart_style#881]
Occurred whilst installing dart_style.

This was a new install:
```
Dart VM version: 2.6.1 (Mon Nov 11 13:12:24 2019 +0100) on "macos_x64"
```

Did a ```pub cache repair``` then a reinstall of dart... Both did not resolve the problem.

```
❯ pub global activate dart_style
Resolving dependencies... (8.3s)
+ _fe_analyzer_shared 1.0.1
+ analyzer 0.39.2
+ args 1.5.2
+ async 2.4.0
+ charcode 1.1.2
+ collection 1.14.12
+ convert 2.1.1
+ crypto 2.1.4
+ csslib 0.16.1
+ dart_style 1.3.3
+ glob 1.2.0
+ html 0.14.0+3
+ js 0.6.1+1
+ meta 1.1.8
+ node_interop 1.0.3
+ node_io 1.0.1+2
+ package_config 1.1.0
+ path 1.6.4
+ pedantic 1.9.0
+ pub_semver 1.4.2
+ source_span 1.5.5
+ string_scanner 1.0.5
+ term_glyph 1.1.0
+ typed_data 1.1.6
+ watcher 0.9.7+13
+ yaml 2.2.0
Precompiling executables... (4.2s)
Failed to precompile dart_style:format:
Error: Could not resolve the package 'kernel' in 'package:kernel/util/graph.dart'.
.pub-cache/hosted/pub.dartlang.org/analyzer-0.39.2/lib/src/summary2/default_types_builder.dart:15:8: Error: Not found: 'package:kernel/util/graph.dart'
import 'package:kernel/util/graph.dart' show Graph, computeStrongComponents;
       ^
.pub-cache/hosted/pub.dartlang.org/analyzer-0.39.2/lib/src/summary2/default_types_builder.dart:256:39: Error: Type 'Graph' not found.
class _TypeParametersGraph implements Graph<int> {
                                      ^^^^^
.pub-cache/hosted/pub.dartlang.org/analyzer-0.39.2/lib/src/summary2/default_types_builder.dart:256:39: Error: Expected 0 type arguments.
class _TypeParametersGraph implements Graph<int> {
                                      ^
.pub-cache/hosted/pub.dartlang.org/analyzer-0.39.2/lib/src/summary2/default_types_builder.dart:125:29: Error: The method 'computeStrongComponents' isn't defined for the class 'DefaultTypesBuilder'.
 - 'DefaultTypesBuilder' is from 'package:analyzer/src/summary2/default_types_builder.dart' ('.pub-cache/hosted/pub.dartlang.org/analyzer-0.39.2/lib/src/summary2/default_types_builder.dart').
Try correcting the name to the name of an existing method, or defining a method named 'computeStrongComponents'.
    var stronglyConnected = computeStrongComponents(graph);
                            ^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by rayk** [comment]
This could be resolved by https://github.com/dart-lang/sdk/commit/ffd757640d82d14bbd44b675feb4ea14ec5bc703

**Comment by rayk** [comment]
This issues is fixed by ```analyzer 0.39.2+1``` .
